### PR TITLE
add e2fsprogs

### PR DESCRIPTION
**Pull request by hoewweken**
Add e2fsprogs
This is a requirement for fuse-ext2

### CODE REVIEW

File: components/sysutils/e2fsprogs/Makefile [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+COMPONENT_VERSION=	1.46.4
+#COMPONENT_REVISION=	1
+COMPONENT_SUMMARY=	Second extended file system (ext2fs) management programs
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH= \
+	sha256:b11042533c1b1dcf17512f0da48e05b0c573dada1dd8b762864d10f4dc399713
+# 1.46.2: 23aa093295c94e71ef1be490c4004871c5b01d216a8cb4d111fa6c0aac354168	
+# 1.46.4: b11042533c1b1dcf17512f0da48e05b0c573dada1dd8b762864d10f4dc399713
+COMPONENT_ARCHIVE_URL= \
```
Comment (AndWac) [inline components/sysutils/e2fsprogs/Makefile:26]: Why are you using a manual "version control"? Please don't leave commented out lines without good reasons and some additional comments.

File: components/sysutils/e2fsprogs/Makefile [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH= \
+	sha256:b11042533c1b1dcf17512f0da48e05b0c573dada1dd8b762864d10f4dc399713
+COMPONENT_ARCHIVE_URL= \
+	https://www.kernel.org/pub/linux/kernel/people/tytso/e2fsprogs/v$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_PROJECT_URL=	http://e2fsprogs.sourceforge.net/
+COMPONENT_FMRI=		sysutils/e2fsprogs
+COMPONENT_CLASSIFICATION= Applications/System Utilities
+COMPONENT_LICENSE=	gnu-gpl-v2 AND gnu-lgpl-v2 AND modified-bsd AND mit
+
```
Comment (AndWac) [inline components/sysutils/e2fsprogs/Makefile:32]: Did you invent this notation of mixed licenses or did you see it elsewhere? I only have seen a different notation yet.

**Comment by hoewweken** [comment]
I didn't see the 
 "force-pushed the e2fsprogs-oi branch from 31d70b1 to 831f2ee "
 and did some weird things. 
 No changes to the COMPONENT_LICENSE
Maybe i should drop this and make a new clean PR ?

**Comment by AndWac** [comment]
Better learn git. It is very powerful. You can change the git history almost arbitrarily. It's usually not necessary to "start from scratch". Of course you can do that locally, eg. with "git reset HEAD^".

**Comment by AndWac** [comment]
Regarding licenses: I recommend to look for similar things in the repository, eg.
cd components
find . -name Makefile -exec grep LICENSE {} \\; -print
I recommend this in many situations - most problems have already been solved in the past.

**Comment by hoewweken** [comment]
I will have a look at the license, i am sure it was copy&paste from somewhere.
I need to check that and use the original license statement.

File: components/sysutils/e2fsprogs/Makefile (hunk 2 above)
Comment (AndWac) [reply components/sysutils/e2fsprogs/Makefile:32]: I think you misunderstood my comment about license notation. What I meant was we are already using a notation for multiple licenses like this:
COMPONENT_LICENSE=	GPLv2,GPLv3,LGPLv2,LGPLv2.1
You had used an uncommon variant.

**Pull request merged**